assets/scss/components/_typeahead.scss

```diff
@@ -0,0 +1,26 @@
+.typeahead {
+	position: relative;
+
+	.typeahead-input {
```

Comment:
does this need to be nested to override some other input class?

Reply:
Sadly it needs to be nested to override the bottom margin. `flush--bottom` would also kill the bottom padding, so that was a no-go